src/Sulu/Bundle/WebsiteBundle/Twig/Seo/SeoTwigExtension.php

```diff
@@ -73,10 +83,18 @@ public function renderSeoTags(array $seoExtension, array $content, array $urls,
             $webspaceKey = $webspace->getKey();
         }
 
+        $request = $this->requestStack->getCurrentRequest();
```

Comment:
After a very quick search I would conclude this is not a good idea:

* https://productforums.google.com/d/msg/webmaster-de/ZSyGtS6S938/Uhuyqh_RwsAJ
* http://www.paulirish.com/2010/the-protocol-relative-url/

The first link is about sitemaps, but I could imagine that google handles the links handled in this PR the same way.

So I think we should probably keep this way. Or maybe @c00n84 could drop some of this seo knowledge here :wink: